tests/integration/targets/aci_aep/tasks/main.yml

```diff
@@ -1,5 +1,6 @@
 # Test code for the ACI modules
```

Comment:
Was there a reason for these 5 classes to add test? Or a generic pick?

Reply:
Just a generic pick. :)